### PR TITLE
docs: translate validate inline into malay lang

### PR DESCRIPTION
**Pull request by kamaroolkarim**

### CODE REVIEW

File: src/ms/validation-inline.php [hunk 1]
```diff
@@ -11,112 +11,112 @@
...
-    'exists'               => 'The selected value is invalid.',
-    'file'                 => 'The content must be a file.',
-    'filled'               => 'This field must have a value.',
+    'boolean'              => 'Ruangan ini mesti benar atau salah.',
+    'confirmed'            => 'Pengesahan yang tidak sepadan.',
+    'date'                 => 'Tarikh ini tidak sah.',
+    'date_equals'          => 'Tarikh mesti sama dengan :date.',
+    'date_format'          => 'Format tarikh tidak sepadan dengan :format.',
+    'different'            => 'Nilai ini dan :other mesti berlainan.',
+    'dimensions'           => 'Dimensi imej tidak sah',
```
Comment (caouecs) [inline src/ms/validation-inline.php:38]: Respect the sort please

File: src/ms/validation-inline.php [hunk 2]
```diff
@@ -11,112 +11,112 @@
...
+        'array'   => 'Saiz jujukan mesti mengandungi :size perkara.',
     ],
-    'starts_with'          => 'This must start with one of the following: :values.',
-    'string'               => 'This must be a string.',
-    'timezone'             => 'This must be a valid zone.',
-    'unique'               => 'This has already been taken.',
-    'uploaded'             => 'This failed to upload.',
-    'url'                  => 'This format is invalid.',
-    'uuid'                 => 'This must be a valid UUID.',
+    'starts_with'          => 'Nilai mesti bermula dengan salah satu dari: :values',
```
Comment (caouecs) [inline src/ms/validation-inline.php:113]: A "." at the end

File: src/ms/validation-inline.php [hunk 3]
```diff
@@ -127,9 +127,9 @@
     | convention "attribute.rule" to name the lines. This makes it quick to
     | specify a specific custom language line for a given attribute rule.
     |
-    */
+     */
```
Comment (caouecs) [inline src/ms/validation-inline.php:130]: No space here

**Review by caouecs — CHANGES_REQUESTED** [review]
Thank you, just some typos

File: src/ms/validation-inline.php [hunk 4]
```diff
@@ -11,112 +11,112 @@
...
-    'after'                => 'This must be a date after :date.',
-    'after_or_equal'       => 'This must be a date after or equal to :date.',
-    'alpha'                => 'This field may only contain letters.',
-    'alpha_dash'           => 'This field may only contain letters, numbers, dashes and underscores.',
-    'alpha_num'            => 'This field may only contain letters and numbers.',
-    'array'                => 'This field must be an array.',
-    'before'               => 'This must be a date before :date.',
-    'before_or_equal'      => 'This must be a date before or equal to :date.',
+    'accepted'             => 'Ruangan ini mesti diterima pakai.',
+    'active_url'           => 'URL ini adalah tidak sah',
```
Comment (andrey-helldar) [inline src/ms/validation-inline.php:17]: A "." at the end.

File: src/ms/validation-inline.php [hunk 5]
```diff
@@ -11,112 +11,112 @@
...
+    'after'                => 'Tarikh mesti tarikh selepas :date.',
+    'after_or_equal'       => 'Tarikh mesti tarikh selepas atau bersamaan :date.',
+    'alpha'                => 'Ruangan ini hanya boleh mengandungi hurug',
```
Comment (andrey-helldar) [inline src/ms/validation-inline.php:20]: A "." at the end.

File: src/ms/validation-inline.php (hunk 1 above)
Comment (andrey-helldar) [inline src/ms/validation-inline.php:38]: A "." at the end.

File: src/ms/validation-inline.php [hunk 6]
```diff
@@ -11,112 +11,112 @@
     | the validator class. Some of these rules have multiple versions such
     | as the size rules. Feel free to tweak each of these messages here.
     |
-    */
+     */
```
Comment (andrey-helldar) [inline src/ms/validation-inline.php:14]: No space here.

**Comment by kamaroolkarim** [comment]
@caouecs @andrey-helldar currently reviewing the comments. will patch it tomorrow.

**Review by andrey-helldar — APPROVED** [review]

**Comment by kamaroolkarim** [comment]
most welcome @caouecs

**Pull request merged**